tiatoolbox/utils/misc.py

```diff
@@ -4,7 +4,7 @@
 import os
 import pathlib
 import zipfile
-from typing import IO, Dict, Optional, Tuple, Union
+from typing import IO, Dict, List, Optional, Tuple, Union
 
```

Comment:
```suggestion
from os import PathLike
```

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/620/commits/f729dc35407b7d347fd39bfb4a24dbe6856d4542